product_docs/docs/pem/9/certificates/index.mdx

```diff
@@ -0,0 +1,312 @@
+- Restarts the PEM server
+
+## Generate a common agent certificate and key pair
+
+You can generate one agent certificate and key pair and use them for all the PEM agents.
+
+1. Create one common agent user in the PEM backend database. Grant `pem_agent` role to the user.
+
+   ```shell
+   # Running as enterprisedb
+   psql -p 5444 -U enterprisedb -d pem
+   CREATE USER pem_agent_common_user;
+   GRANT pem_agent TO pem_agent_common_user;
+   ```
+
+1. Generate an agent key:
+
+   ```shell
+   # Running as root
+   openssl genrsa -out agent.key 4096 
+   ```
+
+1.  Generate a Certificate Signing Request (CSR) for agent:
+
+   ```shell
+   openssl req -new -key agent.key -out agent.csr -subj '/C=IN/ST=MH/L=Pune/O=PEM/CN=<agent_user>'
+   ```
+
+1. Use the openssl x509 command to sign the CSR and generate an agent certificate:
+
+   ```shell
+   openssl x509 -req -days 365 -in agent.csr -CA ca_certificate.crt -CAkey ca_key.key -CAcreateserial -out agent.crt
+   ```
+
+1. Change the permission on the `agent.crt` and `agent.key` file:
+
+   ```shell
+   chmod 600 agent.crt agent.key
+   ```
+
+1. You can use this agent certificate and key pair:
+
+   - For registering the new PEM agent from remote host to the PEM server
+
+     a. Copy the agent certificate and key pair to the remote agent host and register the agent:
+
+        ```shell
+        export PEM_SERVER_PASSWORD=edb
+
+        /usr/edb/pem/agent/bin/pemworker --register-agent \
+        --pem-server 192.168.99.130 \
+        --pem-user enterprisedb \
+        --pem-port 5444 \
+        --pem-agent-user pem_agent_common_user \
+        -o agent_ssl_crt= agent.crt \
+        -o agent_ssl_key= agent.key
+        ```
+
+     b. Enable and start the pemagent services:
+
+        ```shell
+        systemctl enable pemagent
+        systemctl start pemagent
+        ```
+
+   - To replace the agent certificate and key pair with the registered agent:
+ 
+     a. Edit the `agent_user`, `agent_ssl_key`, and `agent_ssl_crt` parameters in `agent.cfg` file of the agent host
+
+        ```shell
+        vi /usr/edb/pem/agent/etc/agent.cfg
+        # Edit the agent username
+        agent_user=pem_agent_common_user
+        # Edit the ssl parameters with new certificate and key file location
+        agent_ssl_key=<new_location>/agent.key
+        agent_ssl_crt=<new_location>/agent.crt
+        ```
+      
+     b. Restart the pemagent service:
+
+        ```shell
+        systemctl restart pemagent
+        ```
+
+
+## Trusted third-party CA certificates and keys
+
+You can replace the PEM self-signed SSL certificates and keys with the trusted third-party CA certificates and keys. 
+
+After obtaining the trusted third-party CA certificates and keys, perform the following steps: 
+
+- Replace PEM self-signed certificates and keys with trusted CA-signed certificates and keys
+- Disable CRL usage by the server if the trusted third-party CA does not provide CRL
```

Comment:
Do we need to provide more information on how to do this? The instructions for the previous bullet refers to the following two subsections, correct? If that is the case and this bullet just describes another task you may need to do, add ", on the server and agents" with server and agents linked to the child sections.

```suggestion
- Disable CRL usage by the server if the trusted third-party CA does not provide CRL
```